cms/templates/partials/metadata-tiles.html

```diff
@@ -7,21 +7,41 @@
           <span class="title">START DATE</span>
           <span class="text">{{ page.product.first_unexpired_run.start_date|date:"F j, Y" }}</span>
           {% if page.product.unexpired_runs|length > 1 %}
-            <span class="dates-link">
-              <a class="dates-tooltip" id="datesPopover" data-trigger="focus" href="#" onClick="event.preventDefault();"
-                data-toggle="popover" data-html="true" data-placement="auto" title="More dates available for this course"
-                data-content="
-                  <div>
-                    <p>Click below to enroll in one of these dates</p>
-                  </div>
-                  {% for course_run in page.product.unexpired_runs %}
+            {% with course_runs=page.product.unexpired_runs %}
+              <span class="dates-link">
+                <a class="dates-tooltip" id="datesPopover" data-trigger="focus" href="#" onClick="event.preventDefault();"
+                  data-toggle="popover" data-html="true" data-placement="auto" title="More dates available for this course"
+                  data-content="
                     <div>
-                      <a class='date-link' href='{% url 'checkout-page' %}?product={{ course_run.products.first.id }}' data-id='{{ course_run.id }}'>Start Date {{ course_run.start_date|date:'F j, Y' }}</a>
+                      <p>Click below to enroll in one of these dates</p>
                     </div>
-                  {% endfor %}">
-                More Dates
-              </a>
-            </span>
+                    {% for course_run in course_runs %}
+                      <div>
+                        <a class='date-link' href='{% url 'checkout-page' %}?product={{ course_run.products.first.id }}'>Start Date {{ course_run.start_date|date:'F j, Y' }}</a>
```

Comment:
why we can't use parameters inside the `url` tag?

Reply:
@tasawernawaz this is  a query string which is not supported by the `url` template tag.